src/main/java/com/pss/handlers/GetProjectileForce.java

```diff
@@ -0,0 +1,30 @@
+package com.pss.handlers;
+import com.pss.interfaces.*;
+
+import javax.vecmath.Vector3d;
+
+public class GetProjectileForce implements IGetProjectileForce{
+    
+    private IGetConfiguration _configurationHandler;
+    private IGetProjectile _projectileHandler;
+
+    public GetProjectileForce(IGetConfiguration configurationHandler, IGetProjectile projectileHandler) {
+        _configurationHandler = configurationHandler;
+        _projectileHandler = projectileHandler;
+    }
+
+    public Vector3d getProjectileForce() {
+        Vector3d additionalForce = _projectileHandler.getProjectile().getVelocity();
+
+        // TODO: possibly refactor this if-check to occur on initialization instead of every frame
```

Comment:
It's always possible for the velocity of the projectile to be 0, so I think we're fine with your current implementation. I think we can remove this comment

Reply:
True true, comment deleted